backend.native/tests/runtime/workers/freeze6.kt

```diff
@@ -38,7 +38,7 @@ fun createInvalidRef1(): FreezableAtomicReference<Any?> {
     val ref = FreezableAtomicReference<Any?>(null)
     ref.value = ref
     ref.freeze()
-    ref.value = null
+
```

Comment:
This test differs from the originally reported one since https://github.com/JetBrains/kotlin-native/pull/3337/commits/869ba1f2bb8f1931e7d752bde182610934e6f4a7. The original test fails again.